pkg/lib/operation/project/local/template/use/operation.go

```diff
@@ -111,6 +111,15 @@ func Run(projectState *project.State, tmpl *template.Template, o Options, d depe
 		return err
 	}
 
+	// Store template information in branch metadata
+	branchState, _ := projectState.Get(o.TargetBranch)
+	targetBranch := branchState.LocalState().(*model.Branch)
+	version := tmpl.Version()
+	if err := targetBranch.Metadata.AddTemplateUsage(instanceId, tmpl.TemplateId(), version.String()); err != nil {
+		return err
+	}
+	branchState.Manifest().(*model.BranchManifest).Metadata = targetBranch.Metadata.ToOrderedMap()
```

Comment:
Reverse-engineeringem jsem přišel na to, že to musí být v metadatech manifestu aby se to dostalo do toho filu, ale vůbec nevím jestli je ok to tam nacpat takhle ručně. 😃  

(Jestli se to tam nemělo dostat nějak automaticky. Ale zkoušel jsem ten persist jak jsi mi radil, ale ten nepomohl. Pokud jsem ho teda měl dobře:
```
if err := projectState.ProjectManifest().PersistRecord(branch state.Manifest()); err != nil {
	return err
}
```)